tests/CMakeLists.txt

```diff
@@ -55,6 +55,12 @@ add_executable(bpftrace_test
   ${CODEGEN_SRC}
 )
 
+add_executable(bpftrace_benchmark
+  benchmarks.cpp
+  mocks.cpp
+  tracepoint_format_parser.cpp
```

Comment:
Would it make sense to extract these 2 TUs out and share with bpftrace_test target? To avoid compiling them twice

Reply:
Sure, I was on the fence. Done.